aya-build/src/lib.rs

```diff
@@ -147,3 +147,21 @@ pub fn build_ebpf(packages: impl IntoIterator<Item = Package>) -> Result<()> {
     }
     Ok(())
 }
+
+#[derive(Default)]
+pub enum Toolchain<'a> {
+    /// Use the latest nightly toolchain to compile to eBPF kernel, e.g. `+nightly`.
```

Comment:
instead of repeating "to eBPF kernel", put this part on the enum's doc comment. also consider calling it EbpfToolchain and drop the word "kernel", it doesn't mean anything int his context.